Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,97 +1,124 @@
 - I helped a team make a course correction based on new information or ideas.
 - I provided appropriate technical freedom, and framing, that allowed the Lead Software Engineers on my teams to thrive independently.
+- I have elevated/shifted my focus significantly because the team(s) I support have been uplifted and empowered to the point they can deliver their mission with me as a sounding board.
 
 </details>
 
 ## 🧭 Culture and Leadership
 
-- I create an environment where others feel safe, by modeling vulnerability, thinking in the open, being wrong, and accepting corrections.
-- I generate energy within the teams I work with, motivating them to achieve great outcomes.
+- I am a proponent of [operational excellence](https://en.wikipedia.org/wiki/Operational_excellence) and [learning from incidents](https://www.learningfromincidents.io/about).
+- I get things right a lot of the time.
+- I create an environment where others feel safe to take risks and make mistakes, by modeling vulnerability, thinking in the open, being wrong sometimes, and accepting corrections.
+- I generate energy in the teams I work with, motivating them to achieve great outcomes.
 - I facilitate effective and timely decision making, especially when the decision affects multiple teams.
 - I build solutions that improve the productivity and/or quality of work of many people/teams.
+- I recognise patterns of organisational dysfunction, by observing data points, amplifying these signals, and advocating effectively for improvement.
+- I am an advisor or contributor to the leadership teams in my group.
 - I set a positive example throughout the company for quality, accountability, and responsibility.
-- I actively share knowledge relevant to product delivery teams.
 - I help define hiring standards and practices.
 
 <details>
 <summary>Examples</summary>
 
-- I drove an entire multiteam program from inception through to shipping code, without regular technical oversight.
+- During a post-incident review, I recognised a valuable lesson and found a way to disseminate the lesson throughout R&D.
+- I drove an entire multiteam program from inception through to shipping code, without regular oversight.
 - I collaborated with other senior leaders to ensure I was aware of all major initiatives at Octopus and could account for them in my own initiatives.
-- I identified a significant problem/opportunity and created a lasting best-fit solution that aligned with our strategy.
+- The teams I work with are consistently winning. They chose ambitious targets, managed scope and expectations effectively, and delivered customer value frequently with customer adoption trends going in the right direction.
+- I identified a problem that was timely, important, and impactful, and worked closely with product, engineering, and executive stakeholders to articulate and refine a solution and delivery plan.
```

Comment:
```suggestion
```

Suggest deleting this example. 

Reasoning: I know you've just moved this in this PR, but this section is now bigger. I don't think this example is concrete enough to be helpful; I find that I'm seeking an "example of the example".